BinaryObjectScanner/Protection/CopyX.cs

```diff
@@ -107,7 +107,7 @@ public IEnumerable<string> CheckDirectoryPath(string path, IEnumerable<string>?
             // intersecting the start of the file.
 
             // Kenny's Adventure uses System instead of ZDAT.
-            string[] dirs = ["ZDAT", "ZDATA", "System"];
+            string[] dirs = ["ZDAT", "ZDATA", "ZZDAT", "ZZDATA", "ZYDAT", "ZYDATA", "System"];
```

Comment:
Would you prefer to sort this in reverse-alphabetical order? A previous comment mentioned "the last directory".

Reply:
Ignore that comment, it's a very shaky assumption. That said, I don't really care what order this all gets sorted in, as long as System gets checked last.

Reply:
The semi-random nature of the list does get to me a little, but as long as it doesn't effectively matter, then that's fine.